crates/compiler/gen_llvm/src/llvm/bitcode.rs

```diff
@@ -206,7 +206,7 @@ fn build_transform_caller_help<'a, 'ctx, 'env>(
     let block = env.builder.get_insert_block().expect("to be in a function");
     let di_location = env.builder.get_current_debug_location().unwrap();
 
-    let arg_type = env.context.i8_type().ptr_type(AddressSpace::Generic);
+    let arg_type = env.context.i8_type().ptr_type(AddressSpace::default());
```

Comment:
the changes to the `gen_llvm/*` files are extremely boring. 

- The `AddressSpace` type is now different, and we can use the `Default` instance now.
- a function for debug info now takes one fewer argument
- some imports became redundant